packages/runtime/src/next/api-handler/handlers/proxy-draft-mode.test.ts

```diff
@@ -0,0 +1,127 @@
+import { NextRequest } from 'next/server'
+import proxyDraftMode from './proxy-draft-mode'
+import { env } from 'process';
+
+jest.mock('next/headers', () => ({
+  draftMode: jest.fn().mockReturnValue({ enable: jest.fn(), disable: jest.fn() }),
+  cookies: jest.fn().mockReturnValue({ get: jest.fn() }),
+}))
```

Comment:
We might want to set these to useful values so that we can test that the appropriate `fetch()` request is being made